vss-manifest.js

```diff
@@ -96,6 +96,26 @@ module.exports = () => {
                     dataType: 'string',
                   },
                 },
+                {
+                  id: 'aquaUrl',
+                  name: 'Aqua Platform URL',
+                  inputMode: 'textbox',
+                  isConfidential: false,
+                  validation: {
+                    isRequired: true,
+                    dataType: 'string',
+                  },
+                },
+                {
+                  id: 'authUrl',
+                  name: 'Aqua Platform Authentication URL',
+                  inputMode: 'textbox',
+                  isConfidential: false,
+                  validation: {
+                    isRequired: true,
```

Comment:
I suppose, this should be not required.

Reply:
the URL is a required field, its region specific for the customers and they need to use the correct value for their tenant